.travis.yml

```diff
@@ -9,33 +11,51 @@ branches:
   only:
     - master
 
-sudo: false
-
-php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-
-env:
-  - WP_VERSION=latest WP_MULTISITE=0
-  - WP_VERSION=latest WP_MULTISITE=1
+cache:
+  - composer
+  - $HOME/.composer/cache
 
 matrix:
   include:
+    - php: 7.1
+      env: WP_VERSION=latest
     - php: 7.0
-      env: WP_VERSION=4.6.4 WP_MULTISITE=1
-
+      env: WP_VERSION=latest
+    - php: 5.6
+      env: WP_VERSION=4.4
+    - php: 5.6
+      env: WP_VERSION=latest
+    - php: 5.6
+      env: WP_VERSION=trunk
+    - php: 5.6
+      env: WP_TRAVISCI=phpcs
```

Comment:
This build can be removed (`PHP 5.6, WP_TRAVISCI=phpcs`). This will run the PHP_Codesniffer, but it will cause the build to fail if there are warnings. I prefer to catch these in CodeClimate, @inderpreet99 your call on if you want to leave this.